frontend/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import './ProgressBar.scss';
+
+interface ProgressBarProps {
+    score: number
+}
+
+export default function ProgressBar(props: ProgressBarProps) {
+
+    const {score} = props;
+
+    return (
+        <div>
+            <progress max="50" value={score}>
```

Comment:
max value at 50? 

what would happen if you have so many applicaiton that i have a score greater than 50? 

I would calculate the value depending on how many job application are in this list and the maximum score per job application.

Reply:
Do I need similar to the score calculation a max calculation? But then it is not comparable, because lists are not equally long and also for the length each lists gains 1 point (= default status rejection).

proposal (3 points is max status for one list item):

const calculateMax = (jobList: IJobList): number => {
        let sum = 0;
        jobList.listItems?.forEach(job => {
            sum = sum + 3
        })
        return sum
    }

Reply:
yeah sounds good. you could optimize the code a bit.

The same thing applies to finding the best list, right now the list with the highest score wins, so a list with a lot of rejection would be better than a list with only a few but succesful job applications. You should have a look if this is what you want 👍

Reply:
Or...
max = JobList.listItems.length x 3 (3 as the maximum possible status number)?